bgpd/bgp_route.c

```diff
@@ -7787,6 +7788,21 @@ void route_vty_out_detail(struct vty *vty, struct bgp *bgp, struct prefix *p,
 					       json_last_update);
 		} else
 			vty_out(vty, "      Last update: %s", ctime(&tbuf));
+
+		/* Line 10 display PMSI tunnel attribute, if present */
+		if (attr->flag & ATTR_FLAG_BIT(BGP_ATTR_PMSI_TUNNEL)) {
```

Comment:
do you need to check attr != NULL here? (I haven't looked at full function)

Reply:
Checked and verified that this is fine, this whole block has the check.